OpenIPSL/Electrical/Controls/package.order

```diff
@@ -2,3 +2,4 @@ CGMES
 PSAT
 PSSE
 Simulink
+VariableSpeedDrive
```

Comment:
The placement of this new sublibrary here breaks with the current structure. Until now we had the different tool implementations. Simply adding `VariableSpeedDrive` on this level would be wrong. @lvanfretti I think we talked about some "generic" models before so how would you group those?

Reply:
@dietmarw and @fachif  I think for things like this "VariableSpeedDrive" we can create a new package under ./Electrical/VSD/Generic